rca/programmes/models.py

```diff
@@ -707,6 +707,20 @@ def get_context(self, request, *args, **kwargs):
         except EmptyPage:
             subpages = paginator.page(paginator.num_pages)
 
+        # Listing filters
+        degree_levels = [i.title for i in DegreeLevel.objects.all()]
```

Comment:
I would also need the key/id for each item so they can be passed over to the API to apply the correct filter.

Reply:
added iID in https://github.com/torchbox/rca-wagtail-2019/pull/274/commits/477264c4c7826f47a1b393ef9f6577bedce46035